PowerEditor/src/WinControls/TreeView/TreeView.h

```diff
@@ -37,7 +37,7 @@ class TreeView : public Window {
 
 	virtual void init(HINSTANCE hInst, HWND parent, int treeViewID);
 	virtual void destroy();
-	HTREEITEM addItem(const TCHAR *itemName, HTREEITEM hParentItem, int iImage, LPARAM lParam = NULL);
+	HTREEITEM addItem(const TCHAR* itemName, HTREEITEM hParentItem, int iImage, LPARAM lParam = 0);

```

Comment:
Semantic problem: it could be `nullptr` but not `0`.

Reply:
`LPARAM` is not a pointer but an integer. So it's not right to compare it against `NULL`. See my previous [comment](https://github.com/notepad-plus-plus/notepad-plus-plus/pull/10048#discussion_r655494587).